certbot/cli.py

```diff
@@ -73,31 +73,32 @@
 COMMAND_OVERVIEW = """The most common SUBCOMMANDS and flags are:
 
 obtain, install, and renew certificates:
-    (default) run   Obtain & install a certificate in your current webserver
-    certonly        Obtain or renew a certificate, but do not install it
-    renew           Renew all previously obtained certificates that are near expiry
-    enhance         Add security enhancements to your existing configuration
-   -d DOMAINS       Comma-separated list of domains to obtain a certificate for
+    (default) run         Obtain & install a certificate in your current webserver
+    certonly              Obtain or renew a certificate, but do not install it
+    renew                 Renew all previously obtained certificates that are near expiry
+    enhance               Add security enhancements to your existing configuration
+   -d DOMAINS             Comma-separated list of domains to obtain a certificate for
 
   %s
-  --standalone      Run a standalone webserver for authentication
+  --standalone            Run a standalone webserver for authentication
   %s
-  --webroot         Place files in a server's webroot folder for authentication
-  --manual          Obtain certificates interactively, or using shell script hooks
+  --webroot               Place files in a server's webroot folder for authentication
+  --manual                Obtain certificates interactively, or using shell script hooks
 
-   -n               Run non-interactively
-  --test-cert       Obtain a test certificate from a staging server
-  --dry-run         Test "renew" or "certonly" without saving any certificates to disk
+   -n                     Run non-interactively
+  --test-cert             Obtain a test certificate from a staging server
+  --dry-run               Test "renew" or "certonly" without saving any certificates to disk
 
 manage certificates:
-    certificates    Display information about certificates you have from Certbot
-    revoke          Revoke a certificate (supply --cert-path)
-    delete          Delete a certificate
+    certificates          Display information about certificates you have from Certbot
+    revoke                Revoke a certificate (supply --cert-path)
+    delete                Delete a certificate
 
 manage your account with Let's Encrypt:
-    register        Create a Let's Encrypt ACME account
-  --agree-tos       Agree to the ACME server's Subscriber Agreement
-   -m EMAIL         Email address for important account notifications
+    register              Create a Let's Encrypt ACME account
+    update_registration   Update a Let's Encrypt ACME account
```

Comment:
In hindsight, I assume that update_registration might be considered uncommon. What are your thoughts?